diff --git a/node_modules/@rhds/elements/elements/rh-cta/rh-cta.js b/node_modules/@rhds/elements/elements/rh-cta/rh-cta.js
index 7bbf242..2455bcd 100644
--- a/node_modules/@rhds/elements/elements/rh-cta/rh-cta.js
+++ b/node_modules/@rhds/elements/elements/rh-cta/rh-cta.js
@@ -6,7 +6,7 @@ import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
 import { DirController } from '../../lib/DirController.js';
 import { colorContextConsumer, colorContextProvider } from '../../lib/context/color.js';
 import { classMap } from 'lit/directives/class-map.js';
-import '@patternfly/pfe-icon';
+// import '@patternfly/pfe-icon';
 import style from "./rh-cta.css.js";
 const supportedTags = ['a', 'button']; // add input later
 function isSupportedContent(el) {
diff --git a/node_modules/@rhds/elements/elements/rh-footer/RhFooter.js b/node_modules/@rhds/elements/elements/rh-footer/RhFooter.js
index e5375f2..aea6cae 100644
--- a/node_modules/@rhds/elements/elements/rh-footer/RhFooter.js
+++ b/node_modules/@rhds/elements/elements/rh-footer/RhFooter.js
@@ -16,7 +16,7 @@ import './rh-footer-social-link.js';
 import './rh-footer-links.js';
 import './rh-footer-block.js';
 import './rh-global-footer.js';
-import '@patternfly/pfe-icon';
+// import '@patternfly/pfe-icon';
 import '@patternfly/pfe-accordion';
 function isHeader(tagName) {
     return !!tagName.match(/^H[1-6]$/i);
diff --git a/node_modules/@rhds/elements/elements/rh-footer/rh-global-footer.js b/node_modules/@rhds/elements/elements/rh-footer/rh-global-footer.js
index 46d657c..9813bc6 100644
--- a/node_modules/@rhds/elements/elements/rh-footer/rh-global-footer.js
+++ b/node_modules/@rhds/elements/elements/rh-footer/rh-global-footer.js
@@ -51,7 +51,7 @@ let RhGlobalFooter = class RhGlobalFooter extends LitElement {
             <slot name="logo">
               <a class="global-logo-anchor"
                   part="logo-anchor"
-                  href="/en"
+                  href="https://www.redhat.com"
                   alt="Visit Red Hat">
                 <svg title="Red Hat logo"
                     class="global-logo-image"
